R/methods-MSnSetList.R

```diff
@@ -78,6 +79,7 @@ setMethod("split", c("MSnSet", "character"),
           
 setMethod("split", c("MSnSet", "factor"),
           function(x, f) {
+              cat("split,MSnSet,factor\n")
```

Comment:
I guess this is a "left-over" from debugging. I think we should remove this.

Reply:
yes. I removed it now